lib/trento/domain/value_objects/saptune_staging.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Trento.Domain.SaptuneStaging do
+  @required_fields [:id]
+
+  use Trento.Type
+
+  deftype do
+    field :enabled, :boolean
+    field :notes, {:array, :string}
+    field :solutions_ids, {:array, :string}
```

Comment:
I would call this `solutions`

Reply:
idk solutions in other value object refer to the whole solution object, I prefer to stick to ids, to just not confuse the stuff 👀

Reply:
well, but you have `notes` above, which is a list of note ids as well...
so we follow the same rule in both entries I guess